test/framework/src/main/java/org/elasticsearch/test/rest/yaml/section/MatchAssertion.java

```diff
@@ -56,6 +56,12 @@ public MatchAssertion(XContentLocation location, String field, Object expectedVa
 
     @Override
     protected void doAssert(Object actualValue, Object expectedValue) {
+        // TODO this needs to be moved to override directory
```

Comment:
++ this needs to be handled by the compat test frame work, but is good here for now.